### ISSUE TITLE
MIC-E format for position-data (to save air-time)

### ISSUE THREAD

**Issue by DB2FM** [lora-aprs/LoRa_APRS_Tracker#35]
It would help to use MIC-E format for the position-data as it would save air-time.

Es wäre schön, wenn das MIC-E Format verwendet würde um die Positionsdaten zu senden, da dies die Sendezeit für jedes Paket verkürzen würde. Den Message-Text könnte man auch nur bei jedem x-ten Paket mit übertragen statt jedes Mal.

**Comment by stroobandt** [comment]
I completely agree. Less bytes with MIC-E would not only save on air time, it would also increase the chances of a correct packet reception, conserve more battery power and decrease the chances on a packet collision with other stations. After all, LoRa has no carrier sense as TNCs do.

Moreover, the `"message": "LoRa Tracker"` consumes 12 extra ASCII bytes and is completely unnecessary and wasteful. Doing so, shows little respect for the way LoRa is intended to work. See also: [Working with Bytes — The Things Network](https://www.thethingsnetwork.org/docs/devices/bytes/).

Advertising of your firmware should be limited to the LoRa APRS iGate.
